libc/misc/unistd_file.c

```diff
@@ -952,3 +1006,468 @@ TEST_GROUP_RUNNER(unistd_file)
 	RUN_TEST_CASE(unistd_file, file_dup2_opened);
 	RUN_TEST_CASE(unistd_file, file_dup2_closed);
 }
+
+
+TEST_GROUP(unistd_file_pread);
+
+
+TEST_SETUP(unistd_file_pread)
+{
+	int fd = open(LOREM_PATH, O_WRONLY | O_CREAT | O_TRUNC, 0777);
+	TEST_ASSERT_GREATER_OR_EQUAL_INT(0, fd);
+	assert_write(fd, LOREM);
+	TEST_ASSERT_EQUAL_INT(0, close(fd));
```

Comment:
This is a setup function - why not just open a file as `O_RDWR` and store `fd` as a global, similar to what was done with regular read/write? I know you test invalid flags and bad fds, but you can also use a second `open()` in such functions and assertions on `open()` in the rest of tests could be left out.